### PR TITLE
turn off feedback and update ms.date

### PR DESCRIPTION
**Pull request by mairaw**
Turning feedback mechanism for the design guidelines since we can't edit that content and updating the ms.date to the date that the book was published.
Also changed the author metadata to use one of the book authors and let ms.author be our default value instead.

### CODE REVIEW

**Comment by BillWagner** [comment]
Do we have the capability to turn off the edit button on a folder level? If so, I'd add that change too.

**Comment by mairaw** [comment]
I'd too but no we can't. I'll request that though internally.

File: docs/standard/design-guidelines/abstractions-abstract-types-and-interfaces.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 ---
 title: "Abstractions (Abstract Types and Interfaces)"
-ms.date: "03/30/2017"
+ms.date: "10/22/2008"
```
Comment (rpetrusha) [inline docs/standard/design-guidelines/abstractions-abstract-types-and-interfaces.md]:
```suggestion
ms.date: "10/22/2008" 
```
is a better date.

File: docs/standard/design-guidelines/serialization.md [hunk 2]
```diff
@@ -79,14 +78,14 @@ Serialization is the process of converting an object into a format that can be r
   
  **✓ DO** make the serialization constructor protected and provide two parameters typed and named exactly as shown in the sample here.  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/standard/design-guidelines/serialization.md]: Good catch!

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I left a suggestion for you to consider. This is ready to merge.

**Pull request merged**